### PR TITLE
Judy: add recipe

### PR DESCRIPTION
**Pull request by kalebo**
Judy is a library for efficient arrays and is a dependency of some scientific software.

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,79 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete the example recipe.

File: recipes/judy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://downloads.sourceforge.net/project/judy/judy/{{name}}-{{version}}/{{name}}-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
Comment (synapticarbors) [inline recipes/judy/meta.yaml]: Please start at build `number: 0`.

File: recipes/judy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  commands:
+    - test -f $PREFIX/lib/libJudy.so  # [linux]
+    - test -f $PREFIX/lib/libJudy.a  # [linux]
+    - test -f $PREFIX/lib/libJudy.dylib  # [osx]
+    - test -f $PREFIX/lib/libJudy.la  # [osx]
+    - test -f $PREFIX/include/Judy.h
+
+about:
+  home: http://judy.sourceforge.net/index.html
+  license: LGPL
```
Comment (synapticarbors) [inline recipes/judy/meta.yaml]: Please add `license_file: COPYING`

Comment (synapticarbors) [inline recipes/judy/meta.yaml]: `license: LGPL-2.1`

**Comment by kalebo** [comment]
@synapticarbors, is there any further changes I should make before someone merges with conda-forge/staged-recipes?

**Comment by mwcraig** [comment]
Thanks @kalebo

**Pull request merged**